app/components/project/CustomMarker.tsx

```diff
@@ -0,0 +1,93 @@
+/* eslint-disable @typescript-eslint/consistent-type-imports */
+import { Link } from "@remix-run/react";
+import { memo, useEffect } from "react";
+import { SlotURI } from "~/types/slotURI";
+import { SVG } from '@svgdotjs/svg.js';
+
+const ICON = `M215.7 499.2C267 435 384 279.4 384 192C384 86 298 0 192 0S0 86 0 192c0 87.4 117 243 168.3 307.2c12.3 15.3 35.1 15.3 47.4 0zM192 128a64 64 0 1 1 0 128 64 64 0 1 1 0-128z`;
+
+interface CustomMarkerProps {
+    item?: SlotURI
+}
+
+function CustomMarker({ item }: CustomMarkerProps) {
+    const id = item?.name.toString().toLowerCase().replace(' ', '_');
+    const image = item?.image_data ? item?.image_data.replaceAll('%23', '#') :
```

Comment:
I have checked locally and there is no image_data in item. The attribute is called image.